ehcache-xml/build.gradle

```diff
@@ -104,8 +104,8 @@ dependencies {
   api 'javax.xml.bind:jaxb-api:[2.2,3)'
   runtimeOnly 'org.glassfish.jaxb:jaxb-runtime:[2.2,3)'
 
-  jakartaApi 'jakarta.xml.bind:jakarta.xml.bind-api:[3,4)'
-  jakartaRuntimeOnly 'org.glassfish.jaxb:jaxb-runtime:[3,4)'
+  jakartaApi 'jakarta.xml.bind:jakarta.xml.bind-api:[3,3.1)'
+  jakartaRuntimeOnly 'org.glassfish.jaxb:jaxb-runtime:[3,3.1)'
```

Comment:
But how would you translate that for the OSGI manifest ?

Reply:
You can't (I don't think) - but for OSGi that's not a big issue since the version range there is for used for "enforcement" purposes only. The bigger issue is what about Maven... since I don't think Gradle can map that reject stanza in to the POM.

Reply:
It may be simpler to stick with `[3,3.1)` even if it's "wrong" just to simplify things.

Reply:
> 

I think so too...